index.js

```diff
@@ -119,7 +119,12 @@ module.exports = function (options) {
    */
 
   return async function userauth(ctx, next) {
-    const loginRequired = !!needLogin(ctx.path, ctx);
+    let loginRequired = null;
+    if(is.asyncFunction(needLogin)){
```

Comment:
please follow the code style.

and need a test case for these change.

Reply:
👌,I will do it later,thks